### ISSUE TITLE
Problems with Ember 2.5.0

### ISSUE THREAD

**Issue by shhQuiet** [kaliber5/ember-bootstrap#79]
Starting a new project and have not used ember-bootstrap before.  I am using ember-cli 2.5.0.

I followed the instructions for installation and had no errors, but I am not getting bootstrap classes, nor the components.  I do see ember-bootstrap in my bower_components folder.

Any help for the newbie is appreciated!

**Comment by simonihmig** [comment]
Are you sure you looked at `bower_components`? ember-bootstrap is an npm module, and should be installed to `node_modules`! Running `ember install ember-bootstrap` should be all that is needed.

It should run with Ember 2.5 out of the box, tests are included as well...

**Comment by shhQuiet** [comment]
Yes, `ember-bootstrap` was in the `bower_components` folder and `bower.json` had `ember-bootstrap` in it as well.  Perhaps I mistakenly ran `bower install ember-bootstrap` instead of `ember install ember-bootstrap`?

I removed `ember-bootstrap` from `bower_components` and removed the dependency in `bower.json`.  Then I ran `ember install ember-bootstrap` and all seems to be working-  Sorry about the bone-headed mistake!  Closing the issue...
